BlobTierAnalysisTool/BlobTierAnalysisTool.NetCore/Helpers/BlobStorageHelper.cs

```diff
@@ -79,28 +51,25 @@ public static bool ParseConnectionString(string connectionString)
         /// </summary>
         /// <param name="containerName"></param>
         /// <returns>True if connecting is validated else false.</returns>
-        public static async Task<bool> ValidateConnection(string containerName = null)
+        public static bool ValidateConnection(string containerName = null)
         {
             try
             {
                 if (string.IsNullOrWhiteSpace(containerName) || containerName == "*")
                 {
-                    await _blobClient.ListContainersSegmentedAsync(null, ContainerListingDetails.None, 1, null, null, null);
+                    blobServiceClient.GetBlobContainers();
                 }
                 else
                 {
-                    var blobContainer = _blobClient.GetContainerReference(containerName);
-                    await blobContainer.ListBlobsSegmentedAsync(null, true, BlobListingDetails.None, 1, null, null, null);
+                    BlobContainerClient blobContainer = blobServiceClient.GetBlobContainerClient(containerName);
+                    blobContainer.GetBlobs();
```

Comment:
Slightly more idiomatic would be to do blobContainer.GetPropertiesAsync. We'd also want to make this an async method.